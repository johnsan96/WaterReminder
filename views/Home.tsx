import { StyleSheet, Text, View } from "react-native";
import { DEV_MODE, YALA_MODE } from "@env";

export default function Home() {

    return (
        <View>
            <Text>Home</Text>
            <Text>yo</Text>
            <Text>Url: {YALA_MODE}</Text> 
        </View>
    );
}
