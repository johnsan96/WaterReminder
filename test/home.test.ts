import Home from "./../views/Home";
import renderer from "react-test-renderer";

/**
 * NOTE:
 * Tests loading from Planning overview.
 */
describe("ObjectIndex View", () => {
  const rendered = renderer.create(Home());

  /**
   * NOTE:
   * tests that component loads
   */
  it("should load", function () {
    expect(rendered.getInstance()).toBeDefined();
  });

  /**
   * NOTE:
   * tests that component matches snapshot struct.
   */
  it("should match the snapshot struct", function () {
    expect(rendered).toMatchSnapshot();
  });
});
