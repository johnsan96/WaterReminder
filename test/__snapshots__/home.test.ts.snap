// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ObjectIndex View should match the snapshot struct 1`] = `
<View>
  <Text>
    Home
  </Text>
  <Text>
    yo
  </Text>
  <Text>
    Url: 
  </Text>
</View>
`;
